### PR TITLE
Small improvements and additions to documentation

### PR DESCRIPTION
**Pull request by gingters**

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -35,49 +36,62 @@ RelayServer deployments consist of multiple instances (pods) of different contai
...
+  **Do NOT** use the example / development project and docker file from this repository in production.  
+  **Do NOT** run multiple instances of the RelayServer.IdentityServer component, as this is not yet supported.
 
   Configuration:
-  * Provide a shared volume to all instances of Thinktecture.Relay.IdentityServer on /var/creds
-  * Provide a connection string to the RelayServer database 
+  * Provide a connection string to the RelayServer database.
 
-* Thinktecture.Relay.Server
+* Thinktecture.Relay.Server
```
Comment (thomashilzendegen) [inline docs/deployment.md:53]: Please remove the trailing white spaces.

File: docs/deployment.md [hunk 2]
```diff
@@ -35,49 +36,62 @@ RelayServer deployments consist of multiple instances (pods) of different contai
...
+  **Do NOT** use the example / development project and docker file from this repository in production.  
+
   Configuration:
   * Provide a connection string to the RelayServer database 
 
 * Thinktecture.Relay.StatisticsApi  
   This component provides endpoints to retrieve usage data about the system from the database. For example you could
   fetch this data in intervals and feed it into your monitoring or billing systems. Load on this component is based
-  on your internal usage only and should be fairly constant.
+  on your internal usage only and should be fairly constant.
```
Comment (thomashilzendegen) [inline docs/deployment.md:87]: Please remove the trailing white spaces.

**Review by thomashilzendegen — CHANGES_REQUESTED** [review]

File: docs/deployment.md (hunk 1 above)
Comment (gingters) [reply docs/deployment.md:53]: This is correct markdown rendering syntax to enforce a line-break,

With trailing whitespace:
image

Without trailing whitespace:
image

File: docs/deployment.md (hunk 2 above)
Comment (gingters) [reply docs/deployment.md:87]: Same thing, this is needed to enforce a line-break in the rendering.

**Review by thomashilzendegen — APPROVED** [review]

**Pull request merged**